### PR TITLE
Added a simple example for Appwrite's Locale API. 

### PR DESCRIPTION
**Pull request by k-guy-dev**
This completes the "Creating a Locale API Vue example" on issue #546 (https://github.com/appwrite/appwrite/issues/546)

### CODE REVIEW

File: example-locale_api/README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+# Appwrite + VueJS =❤️
+
+This example is to showcase [Appwrite's Locale API](https://appwrite.io/docs/client/locale) with [Vue](https://vuejs.org/) by creating a simple web-app that displays your IP, Country,Continent,CountryCode,Continent and Currency.
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
This example is to showcase [Appwrite's Locale API](https://appwrite.io/docs/client/locale) with [Vue](https://vuejs.org/) by creating a simple web-app that displays your IP, Country, Continent, CountryCode, Continent and Currency.
```

File: example-locale_api/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+## Getting Started
+
+To get started quickly we will use vue cli to create the boilerplate that our code will be built on.
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]: Add some more punctuation in this line please

File: example-locale_api/README.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+### Installation
+
+```shell
+npm i -g @vue/cli
+vue create appwrite-locale-vue
+cd appwrite-locale-vue
+```
+
+While we are in the CLI we will also install the Appwrite Web SDK by running:
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
While we are in the CLI, we will also install the Appwrite Web SDK by running:
```

File: example-locale_api/README.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+```shell
+npm install appwrite
+```
+
+and finally we will launch the React development server with:
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]: This line should start with an uppercase letter
```suggestion
And finally we will launch the React development server with:
```

File: example-locale_api/README.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+```shell
+npm run serve -- --port [port_number_of_api_endpoint]
+```
+
+## Introducing the Appwrite SDK
+
+We'll be working with a single component, that will be the app component itself. But first create a `utils.js` file in which we'll be intializing the sdk. Create `utils.js` in the `src` folder and paste the following code.
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]: These sentences need to be looked over again.

File: example-locale_api/README.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+sdk
+  .setEndpoint("[API ENDPOINT]") // Your API Endpoint
+  .setProject("[Your Project ID]"); // Your project ID
+
+export {sdk};
+```
+
+A deeper inspection of this code can be found in the comments within it,
+
+TL:DR: Create a appwrite SDK Instance and initalise it with the endpoint and ProjectID of the project we are working with then export this for usage outside of the file.
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
TL:DR: Create a Appwrite SDK Instance and initalise it with the endpoint and ProjectID of the project we are working with. Then export this for usage outside of the file.
```

File: example-locale_api/README.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+.info h1 {
+  padding-top: 50px;
+}
+</style>
+```
+
+## Explanation of above code:
+
+We first initialized the SDK in our `utils.js` file and then imported the locale.get() method into our `App.vue` file.
+Inside our methods section, we then by using a promise flooded the `info` data with the API's response.
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]: That last past of the sentence is somewhat weird in my opinion.

File: example-locale_api/README.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+We then passed this response to our DOM and displayed the details.😉
+
+Congratulations!!! We just use Locale API to get the clients IP and other details!! Woo-hoo! 🥳🥳🥳
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
Congratulations!!! We just used the Locale API to get the clients IP and other details!! Woo-hoo! 🥳🥳🥳
```

File: example-locale_api/README.md [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+Now you can use these details that you've just got from the Appwrite's Locale API in your own project and personalize every user's experience.
+
+Good Luck! If you need any help feel free to join the [Discord](https://discord.gg/ZFwqr3S) or Refer to the [Appwrite Documentation](https://appwrite.io/docs).
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
Good luck! If you need any help, feel free to join the [Discord](https://appwrite.io/discord) or refer to the [Appwrite Documentation](https://appwrite.io/docs).
```

**Review by stijnvdkolk — CHANGES_REQUESTED** [review]

File: example-locale_api/README.md (hunk 7 above)
Comment (k-guy-dev) [reply example-locale_api/README.md]: @Kolkies , I've changed that part, can you please review it again. Thanks

File: example-locale_api/README.md (hunk 2 above)
Comment (k-guy-dev) [reply example-locale_api/README.md]: Done.

File: example-locale_api/README.md (hunk 5 above)
Comment (k-guy-dev) [reply example-locale_api/README.md]: Made some changes, can you review it ?

File: example-locale_api/README.md [hunk 10]
```diff
@@ -0,0 +1,154 @@
+# Appwrite + VueJS =❤️
+
+This example is to showcase [Appwrite's Locale API](https://appwrite.io/docs/client/locale) with [Vue](https://vuejs.org/) by creating a simple web-app that displays your IP, Country, Continent, CountryCode, Continent and Currency.
+
+## Getting Started
+
+We'll be using vue cli, to create a boilerplate and quickly get started with our project.
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
We'll be using the Vue CLI, to create a boilerplate and quickly get started with our project.
```

File: example-locale_api/README.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+
+And finally we will launch the React development server with:
+
+```shell
+npm run serve -- --port [port_number_of_api_endpoint]
+```
+
+## Introducing the Appwrite SDK
+
+To initialize the Appwrite's SDK, create a `util.js` file in the `src` folder and paste the following code. Here, inside the `setEndpoint` and `setProject` parantheses, add your API endpoint and project ID from appwrite's dashboard.
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
To initialize the Appwrite's SDK, create a `util.js` file in the `src` folder and paste the following code. Here, inside the `setEndpoint` and `setProject` parantheses, add your API endpoint and project ID from Appwrite's dashboard.
```

**Review by stijnvdkolk — CHANGES_REQUESTED** [review]

**Comment by k-guy-dev** [comment]
@Kolkies really sorry for leaving small mistakes. Committed the changes.

**Comment by stijnvdkolk** [comment]
It's ok :D

File: example-locale_api/README.md [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+
+And finally we will launch the React development server with:
+
+```shell
+npm run serve -- --port [port_number_of_api_endpoint]
+```
+
+## Introducing the Appwrite SDK
+
+To initialize the Appwrite's SDK, create a `util.js` file in the `src` folder and paste the following code. Here, inside the `setEndpoint` and `setProject` parantheses, add your API endpoint and project ID from Appwrite's dashboard.
```
Comment (stijnvdkolk) [inline example-locale_api/README.md]:
```suggestion
To initialize the Appwrite's SDK, create a `utils.js` file in the `src` folder and paste the following code. Here, inside the `setEndpoint` and `setProject` parantheses, add your API endpoint and project ID from Appwrite's dashboard.
```

**Review by stijnvdkolk — CHANGES_REQUESTED** [review]
Should be the last change

**Review by stijnvdkolk — APPROVED** [review]

**Comment by k-guy-dev** [comment]
Committed the last change as well. I think it can be merged now.

**Comment by stijnvdkolk** [comment]
I can't merge it, @eldadfux can you do that?

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

@k-guy-dev, please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**